drivers/watchdog/wdt_sam0.c

```diff
@@ -12,8 +12,6 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(wdt_sam0);
 
-#define WDT_REGS ((Wdt *)DT_WDT_SAM0_BASE_ADDRESS)
```

Comment:
Please keep using the base address from dts rather than from ASF. We are doing this by convention for all the vendors

Reply:
ACK, changed.